be/src/olap/column_predicate.h

```diff
@@ -166,13 +166,13 @@ class ColumnPredicate {
     }
     uint32_t column_id() const { return _column_id; }
 
-    virtual std::string debug_string() {
+    virtual std::string debug_string() const {
         return _debug_string() + ", column_id=" + std::to_string(_column_id) +
```

Comment:
I think `column_id` doesn't make sense for users. It's better to get the real column name according to `SegmentIterator` 's variable `_schema`

Reply:
column name maybe removed in the future....